packages/analytics/amplify_analytics_pinpoint_dart/test/string_database_test.dart

```diff
@@ -0,0 +1,197 @@
+// Web only - database regeneration causes multiple Drift database open warning
+
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+@TestOn('browser')
```

Comment:
Why just on web?

Reply:
Rewrote tests and refactored logic to allow for non web.  Before it was due to memory caching changes and later due to issue with db common executor.